serverAkkaHttp/src/main/scala/RouterDerivation.scala

```diff
@@ -16,26 +16,25 @@ trait RouterDerivationModule extends PathMacro with MetaDataMacro with TypePathM
 
 object RouterDerivationMacro extends RouterDerivationModule {
   //val is required to make universe public
-  class MacroHelper[U <: Universe](val universe: U) {
```

Comment:
Passing `universe` like this made the compiler warn about the use of existential types.

I haven't investigated much, but my refactor fixes this and it's roughly the same.